packages/fast-usdc/src/fast-usdc.start.js

```diff
@@ -236,6 +236,11 @@ export const startFastUSDC = async (
 
   produceInstance.reset();
   produceInstance.resolve(instance);
+
+  if ('uusdc' in assetInfo) {
```

Comment:
This if block is to prevent us from calling `connectToNoble` in the a3p environment? If so, it might conflict with the suggestion to include `chainName: 'noble'` assets in `assetInfo`.

Maybe we should have a `builderOpt` instead, where the caller needs to opt-out?

Reply:
good idea; I'll try that...